disnake/ext/commands/bot.py

```diff
@@ -25,13 +25,32 @@
 
 from __future__ import annotations
 
-from typing import Any
+from typing import TYPE_CHECKING, Any, Callable, List, Optional, Sequence, Set, Union
 
 import disnake
 
 from .bot_base import BotBase, when_mentioned, when_mentioned_or
 from .interaction_bot_base import InteractionBotBase
 
+if TYPE_CHECKING:
+
+    import asyncio
+
+    import aiohttp
+    from typing_extensions import Self
+
+    from disnake.activity import BaseActivity
+    from disnake.enums import Status
+    from disnake.flags import Intents, MemberCacheFlags
+    from disnake.i18n import LocalizationProtocol
+    from disnake.mentions import AllowedMentions
+    from disnake.message import Message
+
+    from . import bot_base
```

Comment:
Small nit, importing this as `from .bot_base import PrefixType` would be preferable

Reply:
Oh right, force of habit. Fixed in 843da950de6ee3a671a504ee8019aa8a8640d83a.